Makefile

```diff
@@ -1,39 +1,24 @@
-TGIS_IMAGE=quay.io/opendatahub/text-generation-inference:stable-bafd218
-
-.PHONY: default
+.PHONY: default refresh-poetry-lock-files
 
 default:
-	podman build -t caikit-tgis-serving:$$(git rev-parse --short HEAD) .
-
+	podman build \
+		-t opendatahub/caikit:latest \
+		-t opendatahub/caikit:$$(git rev-parse --short HEAD) \
+		.
 
-.PHONY: refresh-piplock-files
 
-refresh-piplock-files:
-	podman run --user root -it \
-		--rm -v $$(pwd)/caikit:/app/caikit:z \
-		$(TGIS_IMAGE) \
+refresh-poetry-lock-files:
```

Comment:
This target will fail if `opendatahub/caikit:latest` is not in the user's local repo.
```suggestion
# This target requires that the opendatahub/caikit:latest is available in your local registry.
#  you may execute podman pull opendatahub/caikit:latest or make default. 
refresh-poetry-lock-files:
```

Reply:
Forgot to add `: default` dependency. Nice catch